classes/pdf/HTMLTemplateInvoice.php

```diff
@@ -80,7 +88,7 @@ public function getHeader()
     /**
      * Compute layout elements size.
      *
-     * @param $params Array Layout elements
+     * @param array $params Layout elements
```

Comment:
Same here, if we're sure it's an array, we should do `computeLayout(array $params)`

Reply:
It's sure, so I can change to `computeLayout(array $params)` or it's a BC ?
Change only phpdoc don't change behavior

Reply:
Having a phpdoc different than the signature is weird.